week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -13,7 +13,7 @@ function mood() {
 
 function greaterThan10() {
   let num = 10;
-  let isBigEnough;
+  let isBigEnough = 1;
```

Comment:
Cool cheat ;) The idea here was to assign a conditional in this space (i.e. num >= 10)